app/assets/v2/js/wallet.js

```diff
@@ -4,7 +4,7 @@ const eventWalletReady = new Event('walletReady', {bubbles: true});
 const eventDataWalletReady = new Event('dataWalletReady', {bubbles: true});
 
 if (typeof (web3) != 'undefined') {
-  var web3 =  web3;
+  const web3 = web3;
```

Comment:
You can't use const here because we're referencing it before initialisation, lets try:

```
if (!Object.hasOwnProperty.call(window, 'web3')) {
  window.web3 = null;
}
```

Reply:
i just pushed a commit... thanks

https://github.com/gitcoinco/web/pull/9267/commits/9044ad448bf3ea696477e5caa955ebdbca75184f

Reply:
i just pushed a commit... thanks

https://github.com/gitcoinco/web/pull/9267/commits/9044ad448bf3ea696477e5caa955ebdbca75184f